hardware_interface/source/eps.c

```diff
@@ -190,6 +190,10 @@ void EPS_getHK(eps_instantaneous_telemetry_t *telembuf, eps_startup_telemetry_t
 eps_mode_e get_eps_batt_mode() {
     eps_refresh_instantaneous_telemetry();
     eps_instantaneous_telemetry_t eps = get_eps_instantaneous_telemetry();
+
+    //Adding these in order to get the startup telemetry - should be added elsewhere
+    eps_refresh_startup_telemetry();
+    eps_startup_telemetry_t eps_startup = get_eps_startup_telemetry();

```

Comment:
I'm not sure I follow why this is here

Reply:
They weren't called anywhere else. I've removed them.